modules/apps/journal/journal-web/test/js/modals/ImportDataDefinitionModal.js

```diff
@@ -0,0 +1,212 @@
+
+		expect(body).toHaveAttribute('class', '');
+	});
+
+	it('renders import button in footer', () => {
+		const {queryByText} = container;
+
+		const buttonImport = queryByText('import');
+
+		expect(buttonImport).toBeInTheDocument(true);
+	});
+
+	it(
+		'when exists a file selected,' +
+			' the file name appear in disabled input file',
+		() => {
+			const {queryByText} = container;
+
+			const inputFile = document.getElementsByName('testjsonFile')[0];
+			const inputFileName = document.getElementById('testjsonFile');
+
+			fireEvent.change(inputFile, {
+				target: {files: [file]},
+				value: 'C://downloads/Testing.lar',
+			});
+
+			expect(inputFileName).toHaveAttribute('value', 'Testing.lar');
+			expect(inputFileName).toHaveAttribute('disabled');
+
+			const clearButton = queryByText('clear');
+			fireEvent.click(clearButton);
+		}
+	);
+
+	it(
+		'the clear button dont appear,' +
+			' when not exists value in input file name',
+		() => {
+			const {queryByText} = container;
+
+			const clearButton = queryByText('clear');
+
+			expect(clearButton).not.toBeInTheDocument();
+		}
+	);
+
+	it('the clear button appear when exists value in input file name', () => {
+		const {queryByText} = container;
+
+		const inputFile = document.getElementsByName('testjsonFile')[0];
+		const inputFileName = document.getElementById('testjsonFile');
+
+		fireEvent.change(inputFile, {target: {files: [file]}});
+
+		const clearButton = queryByText('clear');
+
+		expect(inputFileName).toHaveAttribute('value', 'Testing.lar');
+		expect(clearButton).not.toBeNull();
+	});
+
+	it('the click in clear button erases the file', () => {
+		const {getByText} = container;
+
+		const inputFile = document.getElementsByName('testjsonFile')[0];
+		const inputFileName = document.getElementById('testjsonFile');
+
+		fireEvent.change(inputFile, {target: {files: [file]}});
+
+		const clearButton = getByText('clear');
+
+		expect(inputFileName).toHaveAttribute('value', 'Testing.lar');
+		expect(clearButton).not.toBeNull();
+
+		userEvent.click(clearButton);
+
+		expect(inputFileName).toHaveAttribute('value', '');
+	});
+
+	it('the import button is disabled when the input arent filled', async () => {
+		const {queryByText} = container;
+
+		const buttonImport = queryByText('import');
+
+		expect(buttonImport).toHaveAttribute('disabled');
+	});
+
+	it('the import button isnt disabled when the inputs are filled', async () => {
+		const {getByLabelText, getByText} = container;
+
+		const inputName = getByLabelText('name');
+		const inputFile = document.getElementsByName('testjsonFile')[0];
+		const inputFileName = getByLabelText('json-file');
+		const buttonImport = getByText('import');
+
+		fireEvent.change(inputFile, {target: {files: [file]}});
+		userEvent.type(inputName, 'structure test');
+
+		expect(inputFileName).toHaveAttribute('value', 'Testing.lar');
+		expect(inputName).toHaveAttribute('value', 'structure test');
+		expect(buttonImport).not.toHaveAttribute('disabled');
```

Comment:
use [toBeEnabled()](https://github.com/testing-library/jest-dom#tobeenabled) instead of `not.toHaveAttribute('disabled')` to assert enabled fields:
```suggestion
		expect(buttonImport).toBeEnabled();
```